sites/footers/reports/a11y-1.json

```
@@ -0,0 +1,2036 @@
+{
+  "@context": "https://wet-boew.github.io/vocab/context/2023/assessment-report-en.json-ld",
+
+  "@type": [ "earl:Assertion", "acr:AssessmentReport" ],
+
+  "earl:subject": {
+    "@id": "_:subject",
+    "dct:references": "https://universallabs.org/wet/labs/pr2043/sites/footers/footers-en.html",
+    "@type": [
+      "earl:TestSubject",
+      "schema:WebPage"
+    ],
+    "dct:description": "The redesigned 2022 site footer that do include an optional contextual footer",
+    "earl:pointer": {
+      "@type": "oa:CssSelector",
+      "@value": "footer#wb-info"
+    }
+  },
+
+  "earl:assertedBy": {
+    "foaf:name": "Service Canada - Principal Publisher",
+    "foaf:homepage": "https://github.com/ServiceCanada",
+    "@type": ["earl:Assertor", "foaf:Organization"],
+    "earl:mainAssertor": {
+      "foaf:name": "Brahim Mahadi Wachilli (Github: @BrahimMahadi)",
+      "foaf:homepage": "https://github.com/BrahimMahadi",
+      "@type": ["earl:Assertor", "foaf:Person"],
+      "foaf:member": {
+    "@id": "_:accessibility_insights",
+        "doap:name": "Accessibility Insights for Web",
+        "doap:homepage": "https://accessibilityinsights.io/",
+        "doap:repository": "https://github.com/microsoft/accessibility-insights-web",
+        "doap:created": "2022-09-30",
+        "doap:release": "v2.35.0",
+        "@type": ["earl:Assertor", "earl:Software"]
+      }
+    }
+  },
+
+  "dct:date": "2022-12-02",
+
+  "dct:description": "Partial evaluation focused only on the component not its container. Evaluated all Success Criteria at Level AA of WCAG 2.2",
+
+  "acr:standard": "https://www.w3.org/TR/WCAG21",
+  "acr:conformanceOption": "act:standard/profiles/wcag#levelAA",
+
+  "earl:result": [
+    {
+      "@id": "_:assessment_tool_result",
+      "earl:test": "act:rules/automated/accessibility_insights_web",
+      "earl:subject": "_:subject",
+    "earl:assertedBy": "_:accessibility_insights",
+      "earl:outcome": "earl:passed",
+      "dct:description": "",
+      "earl:mode": "earl:automatic",
+    "acr:asset": {
+        "@type": "acr:AttachementJson",
```

Typo de ma pars lorsque j'ai fait des précédents request changes.

```suggestion
        "@type": "acr:AttachmentJson",
```

Note: On va pouvoir afficher le contenu JSON après que le PR https://github.com/wet-boew/wet-boew/pull/9631 sera mergé et incorporé à GCWeb. Et ainsi qu'après que le template ai été mise à jour dans GCWeb afin de supporter l'affichage des attachements du type JSON.

---

Changes made in the commit "Fixed typos and broken links".